datasets/hk/legco/crawler.py

```diff
@@ -0,0 +1,186 @@
+"""
+Crawler for extracting names and constituencies of Hong Kong
+Legislative Council members.
+"""
+
+from typing import Dict, Union, NamedTuple, Any
+
+from zavod import Context
+from zavod import helpers as h
+from zavod.logic.pep import categorise
+
+
+MEMBER_URL_FORMAT = (
+    "https://app4.legco.gov.hk/mapi/{lang}/api/LASS/getMember?member_id={member_id}"
+)
+
+
+class MemberPages(NamedTuple):
+    """Member pages in English, Traditional and Simplified Chinese"""
+
+    en: Dict[str, Any]
+    hant: Dict[str, Any]
+    hans: Dict[str, Any]
+
+
+def crawl_member_pages(context: Context, member_id: int) -> MemberPages:
+    # Retrieve member pages in English, traditional and simplified Chinese
+    pages = []
+    for lang in "en", "tc", "cn":
+        member_url = MEMBER_URL_FORMAT.format(lang=lang, member_id=member_id)
+        response = context.fetch_json(member_url, cache_days=7)
```

Comment:
I reckon you can change the caching to 1 day unless it's particularly heavy. It's quite fun to be able to have new people added/departing people dropped the next day

Reply:
Makes sense!  Done.